__tests__/integration/create-token.test.js

```diff
@@ -484,4 +484,42 @@ describe('create token', () => {
 
     done();
   });
+
+  it('create token with data outputs', async done => {
+    const response = await TestUtils.request
+      .post('/wallet/create-token')
+      .send({
+        name: 'Token C',
+        symbol: 'TKC',
+        amount: 100,
+        data: ['test1', 'test2']
+      })
+      .set({ 'x-wallet-id': wallet2.walletId });
+
+    const transaction = response.body;
+    expect(transaction.success).toBe(true);
+
+    await TestUtils.waitForTxReceived(wallet2.walletId, response.body.hash);
+
+    // Make sure the token was minted
+    const tkBalance = await wallet2.getBalance(transaction.hash);
+    expect(tkBalance.available).toBe(100); // The newly minted tokens
+
+    // Make sure the last 2 outputs are the data outputs
```

Comment:
The ordering is guaranteed?
I think that the order can be random if there is a change output

Reply:
Actually it's not random. We shuffle the outputs only when we use the `sendManyOutputsTransaction`. All the others create the transaction object and just mine/push, so we don't shuffle the outputs.